composer.json

```diff
@@ -22,15 +22,22 @@
     "require": {
         "php": "^7.1",
         "doctrine/data-fixtures": "^1.3",
-        "doctrine/doctrine-bundle": "^1.11|^2.0",
+        "doctrine/doctrine-bundle": "^1.11 || ^2.0",
         "doctrine/orm": "^2.6.0",
-        "symfony/doctrine-bridge": "^3.4|^4.1|^5.0",
-        "symfony/framework-bundle": "^3.4|^4.1|^5.0"
+        "symfony/doctrine-bridge": "^3.4 || ^4.1 || ^5.0",
+        "symfony/config": "^3.4 || ^4.3 || ^5.0",
+        "symfony/console": "^3.4 || ^4.3 || ^5.0",
+        "symfony/dependency-injection": "^3.4 || ^4.3 || ^5.0",
+        "symfony/http-kernel": "^3.4 || ^4.3 || ^5.0"
     },
     "require-dev": {
         "doctrine/coding-standard": "^6.0",
         "phpunit/phpunit": "^7.4",
-        "symfony/phpunit-bridge": "^4.1|^5.0"
+        "symfony/phpunit-bridge": "^4.1 || ^5.0",
+        "symfony/framework-bundle": "^3.4 || ^4.0 || ^5.0"
+    },
+    "suggest": {
+        "doctrine/dbal": "To be able to use sharded connections"
```

Comment:
that's useless. As you depend on the ORM, which itself uses DBAL, this suggestion will never be displayed as the package will already be installed.

Thus, this suggestion is wrong. There won't be any improved feature of this bundle when installing DBAL (which is what suggestions are for). The code dealing with sharded connection in FixturesBundle is making sure that the bundle works if you use sharded connections in DoctrineBundle and DBAL. So there is nothing to suggest here.

Reply:
Yep wasn't sure if it was pertinent. @alcaeus told me that the plan was to split the bundle into the dbal bundle and the orm bundle, so maybe it will make sense to add it back as a suggestion when this happens.

I'll remove the suggestion for now.

Reply:
even if we split, that won't ever be pertinent:

- splitting DoctrineBundle in 2 won't mean that the ORM will stop depending on DBAL
- the ShardedConnection logic in FixturesBundle is still not about enabling more features in FixturesBundle but about being compatible with the DBAL/DoctrineBundle features.